lib/cmd/import.js

```diff
@@ -128,15 +126,39 @@ function importSingleUrl(url, prefix, argv) {
   return clayInput.importUrl(url, argv.concurrency)
     .flatMap(chunks.replacePrefixes(prefix))
     .stopOnError(fatalError) // exit early if there's a problem reaching the input clay instance
-    .flatMap((item) => rest.put(item, key, argv.concurrency))
-    .map(showProgress(argv))
-    .toArray(showCompleted);
+    .flatMap((item) => rest.put(item, key, argv.concurrency));
 }
 
-function importSite() {
-  logger.error('Site import not supported yet!');
+/**
+ * import data from a site
+ * @param {string} site
+ * @param {string} prefix of site to import into
+ * @param {object} argv
+ * @return {Stream}
+ */
+function importSite(site, prefix, argv) {
+  const pagesEndpoint = site + '/pages',
```

Comment:
Ah, this offset won't really work, because the page list doesn't really have ordering. (It's deterministic based on uid, but those uids have different schemes based on where they have been generated, e.g. amphora vs kiln vs scratch importer vs claycli vs bootstrap). We should discuss this in person (and I apologize, because I thought I had written out the plan for this somewhere but I can't find it)

The general idea was to query the built-in `pages` elastic index, since that'll have all of the pages in a sortable order (as well as the published status), then base our offset/size off of that.